reference/orders.v2.oas2.yml

```diff
@@ -622,7 +622,7 @@ paths:
         
         Presuming that a valid carrier code is used, a tracking link is generated if either `shipping_provider` or `tracking_carrier` is supplied alongside a tracking number. Providing only the tracking number will result in non-clickable text in the customer facing email.
         
-        Acceptable values for `shipping_provider` include an empty string (`""`), auspost, canadapost, endicia, usps, fedex, royalmail, ups, upsready, upsonline, or shipperhq.
+        Acceptable values for `shipping_provider` include an empty string (`""`), auspost, carrier_{your_carrier_id} (only used if you have set up a carrier using the Shipping Provider API), canadapost, endicia, usps, fedex, royalmail, ups, upsready, upsonline, or shipperhq.
```

Comment:
We should reword this entirely since it doesn't explain anything about `tracking_link` as a writable field.

Reply:
e.g. There are 3 ways to supply a tracking link for shipments:
1. Shipping provider and tracking number
2. Tracking carrier and tracking number
3. Straight up link dawg (paraphrasing)